app/services/bulk_upload/lettings/validator.rb

```diff
@@ -0,0 +1,250 @@
+require "csv"
+
+class BulkUpload::Lettings::Validator
+  include ActiveModel::Validations
+
+  QUESTIONS = {
+    field_1: "What is the letting type?",
```

Comment:
This feels unnecessary (?) Is it different from the questions in the form? Could we get it based on the field id mapping (from `app/services/bulk_upload/lettings/row_parser.rb`)